Articles/Blog/2020-03-maintainers-drive-sustainability.md

```diff
@@ -0,0 +1,112 @@
+
+**What is ParFlow?**
+**Par**allel **Flow** (ParFlow) is an application for integrated hydrology modeling that simulates spatially distributed surface and subsurface flow, as well as land surface processes including evapotranspiration and snow (see Figure 1).  
+
+[Figure 1: ParFlow has been instrumental in understanding how evapotranspiration depletes groundwater under warming and how large scale groundwater development has influence watershed behavior of the contiguous United States. DOIs: 10.1126/sciadv.aav4574, 10.1038/s41467-020-14688-0.]<img src='https://github.com/betterscientificsoftware/images/raw/master/maintainers_image_1.png' class='page lightbox' />
+
+ParFlow simulates the hydrologic cycle from the bedrock to the top of the plant canopy, integrates three-dimensional groundwater flow with overland flow and plant processes using physically-based equations to rigorously simulate fluxes of water and energy in complex real-world systems (see Figure 2). 
+
+[Figure 2: ParFlow is used as the foundation for the Continental United States (CONUS) integrated hydrologic modeling capability.]<img src='https://github.com/betterscientificsoftware/images/raw/master/maintainers_image_2.png' class='page lightbox' />
+
+### Risk Factors for Unsustainable Code
+ParFlow has been in continuous use for scientific discovery since 1992, and several of the current ParFlow developers have been involved with the project since the first equation was written on a white board.  The ParFlow project has undergone long-term development and has been a fertile ground to gain a deep appreciation of the need for solid software engineering practices to improve scientific productivity. 
+
+**Limited software development practices.**  The experiences of ParFlow’s development team matches an [earlier BSSw blog on continuous technology refreshment](https://bssw.io/blog_posts/continuous-technology-refreshment-an-introduction-using-recent-tech-refresh-experiences-on-visit) (CTR), which highlights a core issue - applications that expect to continue to work over time cannot be static, but must undergo ‘*periodic upgrade or replacement of infrastructure to deliver continued reliability, improved speed, capacity, and/or new features*' to respond to the developing needs of users and workflows.  Prior to this project, the ParFlow team had not adopted formal software engineering practices including CTR due to the lack of funding targeted at software maintenance.
```

Comment:
Text says "Prior to this project..."

Which project is that?  IDEAS-Classic?  That's first mentioned much later.  Need change something here.

Reply:
Yes - IDEAS-Classic.
Let's revise to remove "Prior to this project" and start with "The ParFlow team..."

Reply:
Done.  Thanks